Examples/Python/python/acts/examples/reconstruction.py

```diff
@@ -113,9 +113,20 @@
     defaults=[(None, None)],
 )
 
+
 TrackSelectorConfig = namedtuple(
     "TrackSelectorConfig",
-    ["loc0", "loc1", "time", "eta", "absEta", "pt", "phi", "nMeasurementsMin"],
+    [
+        "loc0",
+        "loc1",
+        "time",
+        "eta",
+        "absEta",
+        "pt",
+        "phi",
+        "nMeasurementsMin",
+        "nMeasurementsGroupMin",
+    ],
     defaults=[(None, None)] * 7 + [None],
```

Comment:
```suggestion
    defaults=[(None, None)] * 7 + [None] * 2,
```

@paulgessinger just tested it I think this is missing to make the default construction work

Reply:
Right, I didn't test the default construction!